test/functional/questions_search_controller_test.rb

```diff
@@ -8,15 +8,15 @@ def setup
     activate_authlogic
   end
 
-  test "should get questions_search and render template if question match found" do
-    get :index, id: 'How to'
-    assert_response :success
-    assert_not_nil :title
-    assert_not_nil :tagnames
-    assert_not_nil :users
-    assert_not_nil :notes
-    assert_template :index
-  end
+  #test "should get questions_search and render template if question match found" do
```

Comment:
That controller is marked as temporary until the search system is in place, so perhaps @ananyo2012 and you can coordinate on how to connect up that method? 

https://github.com/publiclab/plots2/blob/master/app/controllers/questions_search_controller.rb#L3-L27